### ISSUE TITLE
Cards UI rule headers should have `aria-controls` attribute per ARIA Accordion pattern

### ISSUE THREAD

**Issue by dbjorge** [microsoft/accessibility-insights-web#2270]
## Describe the bug

We think the collapsible rule sections in our cards UI meet the description of an ARIA Accordion pattern. There is one requirement of that pattern that the cards do not currently fulfill, from https://www.w3.org/TR/wai-aria-practices-1.1/#wai-aria-roles-states-and-properties

> The accordion header button element has `aria-controls` set to the ID of the element containing the accordion panel content.

We think this is low impact (this attribute has [poor AT support](https://a11ysupport.io/tests/tech__aria__aria_controls)), but it would be good to fix it to get in compliance with the standard for the pattern.

**To Reproduce**

1. Open FastPass against any page with errors
2. Open devtools
3. Select one of the `<button>` elements representing a card rule header
4. Observe that it has no `aria-controls` attribute

## Context (please complete the following information)

- OS Name & Version: Any
- AI-Web Version & Environment: Version 2020.3.2.1713 Canary
- Browser Version: Any
- Target Page: Any

**Comment by ferBonnin** [comment]
Leaving this comment for the triage meeting:

As Dan mentions, there is poor support for Aria-controls (only JAWS); I remember reading an article that aria-controls is not great and there is [this open issue](https://github.com/w3c/aria/issues/995) that talks more on the disadvantages in JAWS and that is no longer going to be announced by Jaws.

**Comment by ferBonnin** [comment]
Per conversation with Peter and based on the comment above, we will not add aria-controls
